view/attending_view.py

```diff
@@ -21,7 +22,7 @@ async def callback(self, interaction: discord.Interaction):
             # reset his coop data
             Coop.deregister_user(user)
             # await interaction.response.send_message(f"{user} is not coming!")
-        elif self.label == "Change time":
+        elif self.label == "Set time":
```

Comment:
Imo "Change time" is a bit confusing, "Set time" is more direct if the user wants to set a new time